packages/gatsby-transformer-cloudinary/gatsby-node.js

```diff
@@ -269,7 +269,7 @@ try {
 }
 
 // Is this let just to explain in the docs? And not part of our actual code?
-let globalPluginOptions = {};
+// let globalPluginOptions = {};
```

Comment:
Is this `let globalPluginOptions = {};` just to explain in the docs? And not part of our actual code?

Reply:
Yes, in our case it's set inside `./options.js`